drivers/test.go

```diff
@@ -91,10 +92,10 @@ func init() {
 		Balance: big_spec.Zero(),
 		Code:    builtin_spec.StoragePowerActorCodeID,
 		State: &power_spec.State{
-			TotalNetworkPower:        abi_spec.NewStoragePower(InitialTotalNetworkPower),
-			EscrowTable:              EmptyMapCid,
+			TotalRawBytePower:        abi_spec.NewStoragePower(InitialTotalNetworkPower),
```

Comment:
Can you please file an issue for whatever the reason is that this can't be zero? Where's the panic?

Reply:
I think it can happen here: https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/power/policy.go#L52

After this PR is compilable and works against an impl I will file an issue with details if the problem persists.

Reply:
Confirmed this no longer causes a panic, value is now set to `0`.